lib/cn/csched_sp3_work.c

```diff
@@ -223,13 +223,31 @@ sp3_work_wtype_idle(
     *action = CN_ACTION_COMPACT_KV;
 
     kvsets = cn_ns_kvsets(&tn->tn_ns);
```

Comment:
kvsets should be a uint64_t instead of a uint.

Reply:
guess you already merged. Can you catch it on another PR?

Reply:
The underlying struct member is a uint32_t.  An alternative fix would be to convert `cn_ns_kvsets()` to return a `uint32_t`.

Reply:
nah, that's just silly, it should probably be a uint/uint32_t everywhere.  Consider, at 2K a piece you would need at the very least 4TB of RAM to instantiate 2^32 of them, and probably many multiples of that to support all the additional related structures...

Reply:
`cn_ns_kvsets` returns a `u64` now.  I can change it to `uint32_t` in my open PR.

Reply:
yes, changing to uint32_t would be great